uxarray/io/_icon.py

```diff
@@ -3,16 +3,21 @@
 import numpy as np
 
 
+def _icon_to_ugrid_dims(in_ds):
+    source_dims_dict = {"vertex": "n_node", "edge": "n_edge", "cell": "n_face"}
+    return source_dims_dict
+
+
 def _primal_to_ugrid(in_ds, out_ds):
     """Encodes the Primal Mesh of an ICON Grid into the UGRID conventions."""
-    source_dims_dict = {"vertex": "n_node", "edge": "n_edge", "cell": "n_face"}
+    source_dims_dict = _icon_to_ugrid_dims(in_ds)
 
     # rename dimensions to match ugrid conventions
     in_ds = in_ds.rename_dims(source_dims_dict)
 
     # node coordinates
-    node_lon = np.rad2deg(in_ds["vlon"])
-    node_lat = np.rad2deg(in_ds["vlat"])
+    node_lon = 180.0 * in_ds["vlon"] / np.pi
```

Comment:
@aaronzedwick has already resolved his comment, but this change is probably not even needed, per my [comment](https://github.com/UXARRAY/uxarray/pull/1092/files#r1982281882). Either way is fine with me though